cmd/db-manager/v1beta1/Dockerfile

```diff
@@ -28,4 +28,3 @@ WORKDIR /app
 COPY --from=build-env /bin/grpc_health_probe /bin/
 COPY --from=build-env /go/src/github.com/kubeflow/katib/katib-db-manager /app/
 ENTRYPOINT ["./katib-db-manager"]
-CMD ["-w", "kubernetes"]
```

Comment:
Was this flag not used before?

Reply:
Thanks for the review! @johnugeorge 
Yes, katib-db-manager can not recognize that flag.

```shell
$ docker run --platform linux/amd64 -it docker.io/kubeflowkatib/katib-db-manager -w
flag provided but not defined: -w
Usage of ./katib-db-manager:
```